shared/styles/colors.js

```diff
@@ -16,6 +16,7 @@ const colors = {
   black_50_on_white: 'rgb(127,127,127)',
   black_60: 'rgba(0, 0, 0, 0.60)',
   black_63: 'rgba(0, 0, 0, 0.63)',
+  black_75: 'rgba(0, 0, 0, 0.75)',
```

Comment:
@cecileboucheron is this correct

Reply:
`black` (at 85%) replaced `black_75`.

Reply:
so the zeplin file is not the source of truth? https://app.zeplin.io/project/56b25ffc64fc91773a73aaf8/screen/5bf33ccc2790467ebf9a9bfc

Reply:
No let's remove that line and use `globalColors.black`. We darkened our blackest black and decided to just call it `black`.

Reply:
:white_check_mark: